.circleci/updater_script.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Shouldn't this go in `scripts/`?

Reply:
I can, I just felt it wasn't a script for general use, just mainly for CI.

Reply:
Totally, but it's more like, if I'm looking for a script, that's where I'd look.

Reply:
Makes sense, ill move it and rename it so it's a bit more specific